includes/pipeline.php

```diff
@@ -205,21 +205,16 @@ public static function get($engine, $key) {
         }
 
         $flowData = $data["flowData"];
-
-        try {
-            if ($flowData->{$engine}->{$key}->hasValue) {
-                return $flowData->{$engine}->{$key}->value;
-            }
-            else {
-                error_log($flowData->{$engine}->{$key}->noValueMessage);
-                return null;
-            }
+        
+        if (isset($flowData->{$engine}->{$key}->hasValue)) {
```

Comment:
what if `hasValue` is set, but is `false`?  then you'd probably want to instead error_log a `noValueMessage`?  so the logic might be: 
```
if (isset(hasValue) && hasValue) {
   return value;
} 

if isset(noValueMessage) {
 error_log(noValueMessage)
}
```